python/lsst/obs/base/_fitsRawFormatterBase.py

```diff
@@ -308,7 +308,9 @@ def makeFilterLabel(self):
         filter : `~lsst.afw.image.FilterLabel`
             Object that identifies the filter for this image.
         """
-        return lsst.afw.image.FilterLabel.fromPhysical(self.observationInfo.physical_filter)
+        physical = self.observationInfo.physical_filter
+        band = self.filterDefinitions.physical_to_band[physical]
+        return lsst.afw.image.FilterLabel(physical=physical, band=band)
```

Comment:
Does this work correctly with physical-only or band-only filters? Though I think there may be a requirement that raws can't have the latter.

Reply:
`physical` is unique per instrument per the docstring in `filters.py:143` and the fact that it is a dimension key in the butler.

Reply:
Did you mean to reply to a different comment? If not, I agree, but don't see the relevance...

Reply:
Huh, the comment I meant to reply to does not appear to exist any more.

Reply:
"band-only" does not make sense for raws. The dict above would return `None` for a physical filter that has no band, which preserves the "physical-onlyness" of it.

Reply:
Can you add a test?

Reply:
Just added some unittests.